src/MarkdownTextInput.web.tsx

```diff
@@ -170,15 +172,18 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
     // Empty placeholder would collapse the div, so we need to use zero-width space to prevent it
     const heightSafePlaceholder = useMemo(() => getPlaceholderValue(placeholder), [placeholder]);
 
-    const updateSelection = useCallback(() => {
-      if (!divRef.current) {
-        return;
+    const setEventProps = useCallback((e: NativeSyntheticEvent<any>) => {
+      if (divRef.current) {
+        const text = normalizeValue(divRef.current.innerText || '');
+        if (e.target && typeof e.target !== 'number') {
+          // TODO: change the logic here so every event have value property
+          (e.target as unknown as HTMLInputElement).value = text;
```

Comment:
```suggestion
        if (e.target && typeof e.target !== 'number' && e.target instanceof HTMLInputElement) {
          // TODO: change the logic here so every event have value property
          e.target.value = text;
```

Can we do this instead? You would need to check it first though 😄

Reply:
Yeah, that would be nice, however we want to simulate HTMLInputElement and event target will never be an instance of it. So it doesn't work :(